inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -74,7 +77,8 @@ bool less_by_name(
 
 template <typename T>
 std::string to_str(const T& v) {
-    return std::to_string(v);
+    using std::to_string;
+    return to_string(v);
 }
```

Comment:
Looks like with using you wrote 2 lines instead of previous 1 line. I don't understand the reason of such changes.

Reply:
First line introduce `std::string` as potential overloading (for fundamental types and from std).
Second line will use ADL to find best match (check if class has it own `to_string`).